### PR TITLE
Update install process for rethinkDB

### PR DESCRIPTION
**Pull request by RobertWHurst**
Fixes install process for rethinkDB on travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,9 +2,10 @@ language: node_js
 node_js:
   - "0.10"
 before_install:
-  - sudo add-apt-repository ppa:rethinkdb/ppa -y
-  - sudo apt-get update -qq
-  - sudo apt-get install rethinkdb -y  --force-yes
+  - source /etc/lsb-release && echo "deb http://download.rethinkdb.com/apt $DISTRIB_CODENAME main" | sudo tee /etc/apt/sources.list.d/rethinkdb.list
+  - wget -qO- http://download.rethinkdb.com/apt/pubkey.gpg | sudo apt-key add -
+  - sudo apt-get update
```
Comment (nchelluri) [inline .travis.yml]: Do you want `-qq` here?

File: .travis.yml [hunk 2]
```diff
@@ -2,9 +2,10 @@ language: node_js
...
+  - sudo apt-get install rethinkdb
```
Comment (nchelluri) [inline .travis.yml]: You don't need `-y --force-yes`?

File: .travis.yml (hunk 1 above)
Comment (RobertWHurst) [inline .travis.yml]: @nchelluri It just silences the output from apt, but travis folds up the logs from apt anyway.

File: .travis.yml (hunk 2 above)
Comment (RobertWHurst) [inline .travis.yml]: No, it's not necessary.

**Comment by nchelluri** [comment]
LGTM.

**Pull request merged**